web/src/utils/safeLocalStorage.ts

```diff
@@ -1,5 +1,10 @@
 let hasLocalStorageSupport: boolean | null = null
 
+export const I18N_LANGUAGE_KEY = 'i18nextLng'
+export const PAGE_FORCE_REFRESHED_KEY = 'pageForceRefreshed'
```

Comment:
was changed from `page-force-refresh`, should I keep the old one, delete the new one or just change it as it currently is? leads to two values saved in the local storage